test-cases/scale/longevity-5000-tables-latte.yaml

```diff
@@ -0,0 +1,77 @@
+# test_duration: 1080 # 5h prepare + 10h mixed + 3h margin
+test_duration: 600 # 5h prepare + 2h mixed + 3h margin
+
+n_db_nodes: 6
+instance_type_db: 'i4i.4xlarge'
+n_loaders: 1
+instance_type_loader: 'c6i.4xlarge'
+
+user_prefix: 'schema-scale-5000t'
+append_scylla_yaml:
+  # NOTE: per table metrics on the big count of tables has terrible performance
+  # See: https://github.com/scylladb/scylla-monitoring/issues/2429
+  # So, disable per table metrics
+  enable_node_aggregated_table_metrics: false
+
+# nemesis_class_name: SisyphusMonkey # TODO: uncomment after debugging the 'mixed' phase
+nemesis_class_name: "NoOpMonkey" # TODO: remove after debugging the 'mixed' phase
+nemesis_seed: '777'
+nemesis_interval: 5
+# NOTE: prepare phase fails without nemesis. See https://github.com/scylladb/scylladb/issues/27566
+#       So, it is better to keep prepare phase without nemesis to cover the bug case.
+nemesis_during_prepare: false
+
+round_robin: true
+# NOTE: use 1 latte thread with concurrency=12 to avoid the following bug:
+#       https://github.com/scylladb/scylladb/issues/27551
+prepare_write_cmd:
+  - >-
+    latte run --tag create-schema -f create_schema
+    --threads=1 --concurrency=12 --connections=1
+    --warmup=0 --retries=2 --retry-interval=10s --request-timeout 300 --sampling 1s
+    -d 5000
+    -P keyspace_count=1
+    -P table_count_per_keyspace=5000
+    -P enable_tablets=true
+    -P create_index=false
+    -P replication_factor=3
+    -P compaction_strategy="\"IncrementalCompactionStrategy\""
+    -P sstable_compression="\"LZ4Compressor\""
+    data_dir/latte/schema_scale.rn
+prepare_verify_cmd:
```

Comment:
This phase discloses bugs:
- https://github.com/scylladb/scylladb/issues/27566
- https://github.com/scylladb/scylladb/issues/27620
- https://github.com/scylladb/scylladb/issues/27621